Homework/Develop/index.html

```diff
@@ -22,61 +23,69 @@ <h1>Hori<span class="seo">seo</span>n</h1>
                     <a href="#social-media-marketing">Social Media Marketing</a>
                 </li>
             </ul>
-        </div>
+
+        </nav>
+    </header>
+
+    <div class="hero">
```

Comment:
Research the attribute called `title` and see if you can add a `title` attribute to the `div`. Also, think of why a `title` attribute could be useful for this `div`.

[title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title)

**HINT:** tooltip

Reply:
thanks, looked into this and added the title with an appropriate alt tag here 0ea10bfbe5b3ba05d793c8a6115da7b3169938f4